### DISCUSSION TITLE
Often shows `Loading Failed` when I browser outline

### DISCUSSION THREAD

**Discussion by DreamParadise** [outline/outline#6363] [Self Hosting] [answered]
Error Showing:
![image](https://github.com/outline/outline/assets/3191003/420b9e14-6766-4508-af9e-a94d061a9229)

I can continue browser by click `Reload` button, but this error will appear some later randomly.
Anyone what's the root cause and how can I fix this?

**Comment by tommoor** [comment]
Hi @DreamParadise is this on getoutline.com or self-hosted?

**Reply by DreamParadise** [reply]
Hello, this is self-hosted

**Reply by tommoor** [reply]
In that case it's hard to say. The browser could not access the javascript files it needed to load, you should check network requests in developer tools to see what the network error was.

**Reply by DreamParadise** [reply, marked as answer]
It may be caused by `https-portal` service, I changed to use self-signed certificate now, will come back to comment whether it's fixed some weeks later.
